### ISSUE TITLE
gplinks.co

### ISSUE THREAD

**Issue by sunilroy007** [bogachenko/fuckfuckadblock#206]
### Basic information

**Check the link:** `https://gplinks.co/6Vrps`
**Category:** Breakage
**Reported from:** India

### User environment

**Browser:** mobile kiwi browser version Git210216Gen570536402
**Extension or program:** uBlock Origin 
**DNS:** No

### Additional message

With your script it used to bypass all ads and give me direct link of the file,but now its broken ,its not skipping the gplinks scripts and not giving me actual file,but redirecting me to some news page

**Comment by bogachenko** [comment]
My list doesn't work as the [Universal Bypass](https://github.com/Sainan/Universal-Bypass) extension or as the [AdsBypasser](https://github.com/adsbypasser/adsbypasser) script. The task of the script is to hide/remove annoying pop-ups and notifications about the use of the ad block and he copes with this. My list performs only these two functions, that's all. I doubt that my list in the past could allow you to bypass advertising sites, I think that the site administrator `gplinks.co` I just updated it by adding intermediate advertising sites for earning money. If you want to bypass such advertising sites, check out the repositories that I gave above in this post.
